### PR TITLE
fix(common): Port accessible hide styles from JS to CSS

### PR DESCRIPTION
**Pull request by vibdev**
## Summary

Code to make text screen reader only was added to JS common repo but was not mirrored in the css. This adds a new class to the common css

## Checklist

- [x] branch has been rebased on the latest master commit
- [x] `yarn test` passes
- [x] code has been documented and, if applicable, usage described in README.md
- [x] a11y approved final implementation
- [x] code adheres to the [API & Pattern guidelines](../API_PATTERN_GUIDELINES.md)

## Additional References

This PR also changes Change word-wrap: normal to white-space, see:

https://medium.com/@jessebeach/beware-smushed-off-screen-accessible-text-5952a4c2cbfe

### CODE REVIEW

File: modules/common/css/lib/accessibility.scss [hunk 1]
```diff
@@ -21,3 +21,16 @@
     outline: none;
   }
 }
+
+.wdc-accessible-hide {
```
Comment (anicholls) [inline modules/common/css/lib/accessibility.scss]: Can you turn this into a mixin and call it from within `.wdc-accessible-hide` so it's available as both a mixin and a class?

File: modules/common/react/lib/styles/accessibleHide.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ const accessibleHide: CSSObject = {
   clipPath: 'polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px)',
   position: 'absolute',
   overflow: 'hidden',
-  wordWrap: 'normal',
+  whiteSpace: 'nowrap',
```
Comment (anicholls) [inline modules/common/react/lib/styles/accessibleHide.ts]: What was the reason for this change?

Comment (vibdev) [reply modules/common/react/lib/styles/accessibleHide.ts]: linked a article at top from facebook engineer. 
TLDR: Some browser / screen reader combos don't read the space between words hidden with clip. Basically the words get newline separated and they don't read new lines as a space -> Hello World -> Hello{LineFeed}World
I read a few blogs today to make sure this is still the best method and this seems like the most bullet proof.

File: modules/common/css/lib/accessibility.scss (hunk 1 above)
Comment (vibdev) [reply modules/common/css/lib/accessibility.scss]: updated!
